test/functional/tests/io_class/test_io_classification.py

```diff
@@ -475,6 +477,7 @@ def test_ioclass_id_as_condition(filesystem):
                       "it is classified properly."):
         # Should be classified as IO class 5
         base_occupancy = cache.get_io_class_statistics(io_class_id=5).usage_stats.occupancy
+        time.sleep(1)
```

Comment:
sleep(1) after IO class configure and/or directory structure change is enough

Reply:
Right! Thank you, I've tested it out and it's good:)